app/scripts/controllers/app-state.js

```diff
@@ -56,7 +55,7 @@ export default class AppStateController extends EventEmitter {
       // multiple networks.
       hadAdvancedGasFeesSetPriorToMigration92_3: false,
       ...initState,
-      qrHardware: {},
+      qrHardwarePopoverState: {},
```

Comment:
I was going to comment that we should have a migration for this data, but it looks like it gets discarded when the controller gets constructed :thinking: 

But still, the data is renamed and the old name would remain in state now that it's no longer being cleared here.

Could we keep this named as `qrHardware` perhaps, to avoid the need for a migration? Either that or we could add a migration just to remove this state